<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>365</width>
    <height>575</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>365</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tetris</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LabelGameOver">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/debug/pause.png</normaloff>../build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/debug/pause.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/left.png</normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/left.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/down.png</normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/down.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/right.png</normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/right.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rotationButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/plus.png</normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/plus.png</normaloff>../../../QtProjects/build-tetris-Desktop_Qt_6_2_1_MinGW_64_bit-Debug/res/plus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
